### PR TITLE
[Update] [weave.lic] Changed weave to use hometown setting. 

### PR DESCRIPTION
**Pull request by Dartellum**
Added all information to base-crafting for loom-rooms and made definition to find empty room with loom.

### CODE REVIEW

File: common-crafting.lic [hunk 1]
```diff
@@ -78,6 +78,16 @@ module DRCC
     end
   end
 
+  def find_loom_room(hometown, override = nil)
+    if override
+      DRCT.walk_to(override)
+    else
+      loomrooms = get_data('crafting')['tailoring'][hometown]['loom-rooms']
+      idle_room = get_data('crafting')['tailoring'][hometown]['idle-room']
+      DRCT.find_sorted_empty_room(loomrooms, idle_room)
```
Comment (KatoakDR) [inline common-crafting.lic:87]: Disclaimer, I'm not familiar with this script. If there isn't an idle room configured, should it fall back to your safe room?

Comment (rpherbig) [reply common-crafting.lic:87]: @BinuDR @Dartellum - you two have contributed to this script, what do you think?

File: common-crafting.lic [hunk 2]
```diff
@@ -78,6 +78,16 @@ module DRCC
     end
   end
 
+  def find_loom_room(hometown, override = nil)
+    if override
+      DRCT.walk_to(override)
+    else
+      loomrooms = get_data('crafting')['tailoring'][hometown]['loom-rooms']
```
Comment (rpherbig) [inline common-crafting.lic:85]: `loom_rooms`

File: data/base-crafting.yaml [hunk 3]
```diff
@@ -467,16 +467,20 @@ tailoring:
     - 16661
     - 16663
     sewing-rooms:
-    - 19_037
-    - 19_036
-    - 16_670
+    - 19037
```
Comment (rpherbig) [inline data/base-crafting.yaml:470]: The underscore here is a Rubocop style thing.

Which reminds me, we should probably run Rubocop on the repo sometime soon. I just don't have it/Ruby installed on any machines at the moment.

File: common-crafting.lic (hunk 2 above)
Comment (Dartellum) [reply common-crafting.lic]: I just cut and pasted `def find_shaping_room` and changed `shapeingrooms` to `loomrooms`. Changed!

File: common-crafting.lic (hunk 1 above)
Comment (Dartellum) [reply common-crafting.lic]: We can add an idle room then. I will add the room right outside the weave rooms.

Comment (Dartellum) [reply common-crafting.lic]: Updated.

**Pull request merged**